.npmrc

```diff
@@ -0,0 +1,2 @@
+tag-version-prefix="v"
+save-prefix="~"
```

Comment:
would suggest exact versions rather than any floating version - but your call

Reply:
1. `package.json` already specifies the exact versions, and is what `npm install` prefers (I believe).
2. I actually see exact versions in package.json as an anti-pattern, as it makes it redundant with  `package-lock.json`, while nullifying any value to be had from the `npm outdated` and `npm upgrade` commands.